Main.py

```diff
@@ -0,0 +1,94 @@
+import pygame, sys, time, math, random
+from pygame.locals import *
+
+pygame.init()#Initiates Pygame
+img1 = pygame.image.load('BigSmoke.jpg')  #Imports BigSmoke
+img2 = pygame.image.load('Harambe.jpg')   #Imports Harambe
+img3 = pygame.image.load('Doge.jpg')      #Imports Doge
+img4 = pygame.image.load('YeaBoi.jpg')    #Imports Boi
+img5 = pygame.image.load('Dio.jpg')       #Imports Dio
+img6 = pygame.image.load('Jaguars.jpg')   #Imports Jaguars
+myfont = pygame.font.SysFont("comic", 100) #Imports text font
+Height =800 #Sets dimensions of window
```

Comment:
Use all-capitals (e.g. HEIGHT = 800) for constants. Review the [PEP-8 Guidelines](https://www.python.org/dev/peps/pep-0008/#constants).